bin/runnable-list.js

```diff
@@ -1,40 +1,37 @@
 'use strict'
 
-var program = require('commander')
+const program = require('commander')
 
-var runnable = require('../lib/runnable')
-var Table = require('../lib/table')
-var utils = require('../lib/utils')
+const List = require('../lib/list')
+const Table = require('../lib/table')
+const Utils = require('../lib/utils')
 
 program
   .arguments('[repository]')
   .description('Lists all repositories and services. Specify a repository to list its containers.')
   .parse(process.argv)
 
-var options = {
-  _user: runnable.user,
+const options = {
   repository: program.args.shift()
 }
 
 if (options.repository) {
-  runnable.listContainersForRepository(options)
-    .then(function (results) {
+  List.listContainersForRepository(options)
+    .then((results) => {
       Table.log(results)
     })
-    .catch(utils.handleError)
+    .catch(Utils.handleError)
 } else {
-  runnable.listContainerSummary(options)
-    .then(function (results) {
+  List.listContainerSummary(options)
+    .then((results) => {
       function tableFormatter (obj, cell) {
-        for (var key in obj) {
+        for (const key in obj) {
```

Comment:
I'm not changing this. This is correct and is [supported by the spec](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-for-in-and-for-of-statements-static-semantics-boundnames).

What this prevents me (the developer) from doing is something like

```js
for (const key in obj) {
  key = key + '!' // this is invalid (because it's const)
}
```

That's all that it's doing. The `const` _isn't_ changing. The loop is being run repeatedly with various `keys`, but I'm being strict in where the `key` cannot be mutated.

(this actually is a good question though and definitely made me think)